plugins/source/postgresql/resources/plugin/plugin_test.go

```diff
@@ -137,7 +141,7 @@ func getTestCases(serialValue int64) []testCase {
 		{"circle", "<(1,2),3>", &scalar.String{Value: "<(1,2),3>", Valid: true}},
 		{"date", "1999-01-08", &scalar.Date32{Value: 10599, Valid: true}},
 		{"double precision", 1.1, &scalar.Float{Value: 1.1, Valid: true, BitWidth: 64}},
-		{"inet", "192.168.0.1/24", &inet},
+		{"inet", "192.168.0.0/24", &inet},
```

Comment:
We can also revert this, should work normally

Reply:
Good catch, done in https://github.com/cloudquery/cloudquery/pull/11719/commits/482c8f264089fcd8931fb8cdf0f3d8a50c645eb7